Pinta.Gui.Widgets/Widgets/StatusBarColorPaletteWidget.cs

```diff
@@ -41,10 +42,10 @@ public sealed class StatusBarColorPaletteWidget : Gtk.DrawingArea
 	private RectangleD palette_rect;
 	private RectangleD recent_palette_rect;
 
-	const int PALETTE_ROWS = 2;
-	const int SWATCH_SIZE = 19;
-	const int WIDGET_HEIGHT = 42;
-	const int PALETTE_MARGIN = 10;
+	public const int PALETTE_ROWS = 2;
```

Comment:
Rather than making all of these public (along with methods like `GetSwatchAtLocation()`), it might be cleaner to have a reusable palette widget that can be used both for the status bar and the dialog? Right now things seem tightly coupled / duplicated, with the color picker dialog having to carefully set up itself in the exact same manner as this widget